docs/guides/monitor/pi-hole-raspberry-pi.md

```diff
@@ -0,0 +1,148 @@
+
+## Prerequisites
+
+All you need to get started is a [Raspberry Pi](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/) with
+Raspbian installed. This guide uses a Raspberry Pi 4 Model B and Raspbian GNU/Linux 10 (buster).
+
+This guide assumes you're connecting to a Raspberry Pi remotely over SSH, but you could also complete all these steps on
+the system directly using a keyboard, mouse, and monitor.
+
+## Install Netdata
+
+Let's start by installing Netdata first. This way you'll start collecting system metrics as soon as possible for the
+longest and deepest database of historical metrics.
+
+We recommend installing Netdata with the one-line kickstart script on Raspbian systems. Check out our [install
+docs](/packaging/installer/README.md) if you want to try a different method.
+
+```bash
+bash <(curl -Ss https://my-netdata.io/kickstart.sh)
+```
+
+The installer asks you to install dependencies, then compiles Netdata from source via
+[GitHub](https://github.com/netdata/netdata).
+
+Once installed on a Raspberry Pi 4 with no accessories, Netdata starts collecting roughly 1,500 metrics every second and
+populates its dashboard with more than 250 charts.
+
+Open your browser of choice and navigate to `http://NODE:19999/`, replacing `NODE` with the IP address of your Raspberry
+Pi. Not sure what that IP is? Try running `hostname -I | awk '{print $1}'` from the Pi itself.
+
+You'll see Netdata's dashboard and a few hundred real-time,
+[interactive](https://learn.netdata.cloud/guides/step-by-step/step-02#interact-with-charts) charts. Feel free to
+explore, but let's turn our attention to installing Pi-hole.
+
+## Install Pi-Hole
+
+Like Netdata, Pi-hole has a one-line script for simple installation. From your Raspberry Pi, run the following:
+
+```bash
+curl -sSL https://install.pi-hole.net | bash
+```
+
+The installer will help you set up Pi-hole based on the topology of your network. Once finished, you should set up your
+devices—or your router for system-wide sinkhole protection—to [use Pi-hole as their DNS
+service](https://discourse.pi-hole.net/t/how-do-i-configure-my-devices-to-use-pi-hole-as-their-dns-server/245). You've
+finished setting up Pi-hole at this point.
+
+As far as configuring Netdata to monitor Pi-hole metrics, there's nothing you actually need to do. Netdata's [Pi-hole
+collector](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/pihole) will autodetect the new service
+running on your Raspberry Pi and immediately start collecting metrics every second.
+
+Restart Netdata with `sudo service netdata restart` to spin up the Pi-hole collector again. When you refresh your
+Netdata dashboard or load it up again in a new tab, you'll see a new entry in the menu for **Pi-hole** metrics.
+
+## Use Netdata to explore and monitor your Raspberry Pi and Pi-hole
+
+By the time you've reached this point in the guide, Netdata has already collected hundreds of thousands of metrics
```

Comment:
> hundreds of thousands of metrics

Reply:
I probably should have been clearer that I mean hundreds of thousands of metrics values. 1500 metrics * 10 minutes = many values. :wink:

Reply:
Yes, first that come to my mind is 

(from the dashboard)

```cmd
Every second, Netdata collects 2,197 metrics
```

Reply:
> = many values

They are data points if i remember it right 🤷‍♂️